templates/metrics/css/popularity.html

```diff
@@ -9,6 +9,14 @@
   {% include "metrics/css/_css_metric_nav.html" %}
 {% endblock %}
 
+{% block properties %}
+<nav class="data-panel" style="font-size: smaller;">
+  All Properties:<a href="/metrics/css/timeline/popularity" style="padding-left: 2%; padding-right: 2%;">Timeline</a> | 
```

Comment:
The phrase "All properties" is redundant because it already appears in the heading line just below this nav.  Try "View as:" instead.

Reply:
Done